### ISSUE TITLE
🚀 Feature: ticket feature on navbar is not responding

### ISSUE THREAD

**Issue by pushkarroy** [novuhq/hacksquad-website#149]
### 🔖 Feature description

if you click on the ticket feature then it is not responding.

### 🎤 Why is this feature needed ?

to make the website work properly it's necessary to that every thing work properly

### ✌️ How do you aim to achieve this?

I think if we use an anchor tag where it should go then it will be great

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

None

**Comment by ayushsgithub** [comment]
I don't think so @pushkarroy, I just checked it and it's working fine! If you are having any trouble, please make video where you are getting this error, it will be much easier to solve the problem then.
